spec/index.test.js

```diff
@@ -41,33 +30,57 @@ describe('Database Tests', () => {
     await Photos.create(mockData);
 
     server = await app.listen(4000, (err) => {
-    if (err) {
-     return done(err);
-    }
+      if (err) {
+       return done(err);
+      }
 
-    agent = request.agent(server); // since the application is already listening, it should use the allocated port
-    done();
+      agent = request.agent(server); // since the application is already listening, it should use the allocated port
+      done();
     });
   });
+
   afterAll(async function(done){
     await mongoose.disconnect();
     return server && await server.close(done);
   });
 
-  it('Should retrieve data from Photos database', function(done) {
+  test('Should retrieve the correct amount of dcouments from the database', function(done) {
```

Comment:
Cool idea, but what are we really testing here...

To me, it seems like we're testing whether or not Mongoose's `deleteMany`, `create`, and `countDocuments` methods work, but we can safely assume they've written their own tests.

I think it would be more meaningful if you sent a get request to the server, to test whether or not:

1. The endpoint works (because it's not being tested elsewhere in this file)
2. If you get the proper status code back
3. If you get the right data back (Does the `roomId` of the photo match the `roomId` you requested?)

Reply:
Also, you have a typo in your spelling of "documents" (`dcouments`)

Reply:
We addressed this in person - resolving convo